src/vtbackend/StatusLineBuilder.cpp

```diff
@@ -156,6 +157,7 @@ std::optional<StatusLineDefinitions::Item> makeStatusLineItem(
             styles,
             activeColor,
             activeBackground,
+            std::nullopt,
```

Comment:
can we use designated initializers here?

Reply:
I will add a comment, that this is separator, because this class has an inheritance and we can not use designated initialization here :(